manim/mobject/geometry.py

```diff
@@ -809,3 +782,51 @@ class RoundedRectangle(Rectangle):
     def __init__(self, **kwargs):
         Rectangle.__init__(self, **kwargs)
         self.round_corners(self.corner_radius)
+
+
+class ArrowTip(VMobject):
+    CONFIG = {
+        "fill_opacity": 0,
+        "stroke_width": 0,
+        "length": DEFAULT_ARROW_TIP_LENGTH,
+        "start_angle": PI,
+    }
+
```

Comment:
Suggestion: perhaps add
```python
def __init__(self):
    pass
```
Explicity is better than implicit and whatnot

Reply:
I fully agree. A more agressive suggestion is in bb1746d0b95510cf61e6b4bfe446ef1c55854738 (because I think it is better to avoid even the slightest chance of silent failures).

Reply:
Oh I like your solution. I was about to say this should be an AbstractBaseClass, but that would have meant triple inheritance which I'm not going anywhere near.